notebooks/pathomics/getting_started_with_digital_pathology.ipynb

```diff
@@ -77,7 +77,7 @@
       "outputs": [],
       "source": [
         "%%capture\n",
-        "!pip3 install wsidicom==0.20.5 idc-index==0.6.2 tensorflow==2.15"
+        "!pip3 install wsidicom==0.20.5 idc-index==0.6.3 tensorflow==2.15"
```

Comment:
I suggest we require 0.6.3 or above, but not fix to be equal to this release!

Reply:
But if we require above and change the behavior or even name of multiple indices, the notebook will not work anymore.

Reply:
Right, that's why changing the behavior or name of the indices is not a good idea.

If we fix the version, we will be forever stuck with IDC data release v18. Data release is frozen for a given release of idc-index package.

Reply:
follow-up discussion on Slack

<img width="1011" alt="image" src="https://github.com/user-attachments/assets/78284125-d0ce-419c-814e-6e4410d3dec0">